marketplace_builder/views/pages/use-cases/e-commerce/adding-photos-to-products.liquid

```diff
@@ -0,0 +1,320 @@
+---
+converter: markdown
+metadata:
+  title: Adding Photos to Products
+  description: This guide will help you to add photo management to your products
+slug: use-cases/e-commerce/adding-photos-to-products
+searchable: true
+---
+
+This guide will help you to add photos to your products and is a larger topic of product. Product management consists of three separate sections:
+
+1. Product properties management
+2. **Product images management**  (described in this topic)
+3. Stock levels management
+
+## Requirements
+
+To follow this tutorial, you should be familiar with basic PlatformOS concepts, and the topics in the Get Started section. You should have followed this tutorial series up to the previous part "Creating Product Management Forms", where you’ve set up forms for creating, editing and deleting products.
+
+- [How Platform OS Works](/how-platformos-works)
+- [Get Started](/get-started)
+- [Creating Product Management Forms](/use-cases/e-commerce/creating-product-management-forms)
+- [Uploading Files Directly to Amazon S3 Using AJAX](/tutorials/user-uploads/uploading-files-directly-amazons3-using-ajax)
+
+## Steps
+
+Adding photos to products is a six-step process:
+
+1. Create new photo form
+2. Add upload script
+3. Create photo destroy form
+4. Create query to fetch photos from database
+5. Create photos partial
+6. Modify product details page
+
+
+### Step 1: Create new photo form
+
+Photo form will be included on the product details page. Photo upload is a two-step process. First, you upload image directly to a S3 bucket, then you submit the form itself, including url to the newly uploaded file. You can read more about [Uploading Files Directly to Amazon S3 Using AJAX](/tutorials/user-uploads/uploading-files-directly-amazons3-using-ajax) in our documentation.
+
+#### marketplace_builder/form_configurations/photo/create_photo_form.liquid
+
+```liquid
+{% raw %}
+---
+name: create_photo_form
+resource: Customization
+redirect_to: /admin/product/{{ params.form.properties_attributes.product_id }}
```

Comment:
i think it needs to be in quotes for liquid to work inside yaml key

Reply:
It works just fine when using single line and no special characters. I can add quotes to be on the safe side